Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'activeadmin', github: 'yeti-switch/active_admin'
 gem 'novus-nvd3-rails', github: 'yeti-switch/nvd3-community-rails'
 gem 'active_admin_theme'
 gem 'active_admin_import' , '3.0.0.pre'
+gem 'active_admin_scoped_collection_actions', github: 'Zamyatin-AA/active_admin_scoped_collection_actions', branch: 'batch_action_collection_fix', ref: 'd199e2867d950143a6b2c74e54c3f289d8a9b1e0'
```

Comment:
This fix accepted in gem release 0.3.5. 
`gem 'active_admin_scoped_collection_actions'` should work

`$ bundle update active_admin_scoped_collection_actions` should do the trick for local update.